### DISCUSSION TITLE
Xamarin.Android Leanback using MvvmCross

### DISCUSSION THREAD

**Discussion by isaranghi** [MvvmCross/MvvmCross#4310] [Q&A]
Hello,

I am starting to write a Xamarin.Android application for TV using Leanback library. I am just wondering if there is any sample or tutorial available that shows how to implement Leanback application using Xamarin.Android and MvvmCross.

Thanks.

**Comment by Cheesebaron** [comment]
We only have minimal support for it. We only have bindings for HorizontalGridView, and support for some of the fragment types that leanback has. Haven't worked with TV's myself, so your mileage may vary.
